src/cascadia/UnitTests_TerminalCore/SelectionTest.cpp

```diff
@@ -672,5 +672,121 @@ namespace TerminalCoreUnitTests
+            }
+
+            // Step 2: Shift+Click to "dragThroughHere"
+            {
+                // Simulate Shift+Click at (x,y) = (21,10)
+                //
+                // buffer: doubleClickMe dragThroughHere
+                //         ^                ^
+                //       start            finish
+                term.SetSelectionEnd({ 21, 10 }, ::Terminal::SelectionExpansionMode::Cell);
+
+                // Validate selection area: "doubleClickMe drag" selected
+                ValidateSingleRowSelection(term, SMALL_RECT({ 4, 10, 21, 10 }));
+            }
+
+            // Step 3: Shift+Double-Click to "dragThroughHere"
+            {
+                // Simulate Shift+DoubleClick at (x,y) = (21,10)
+                //
+                // buffer: doubleClickMe dragThroughHere
+                //         ^                ^
+                //       start            finish
+                term.SetSelectionEnd({ 21, 10 }, ::Terminal::SelectionExpansionMode::Word);
+
+                // Validate selection area: "doubleClickMe dragThroughHere" selected
+                ValidateSingleRowSelection(term, SMALL_RECT({ 4, 10, 32, 10 }));
+            }
+
+            // Step 4: Shift+Double-Click to "dragThroughHere"
+            {
+                // Simulate Shift+TripleClick at (x,y) = (21,10)
+                //
+                // buffer: doubleClickMe dragThroughHere
+                //         ^                ^
+                //       start            finish
+                term.SetSelectionEnd({ 21, 10 }, ::Terminal::SelectionExpansionMode::Line);
+
+                // Validate selection area: "doubleClickMe dragThroughHere..." selected
+                ValidateSingleRowSelection(term, SMALL_RECT({ 4, 10, 99, 10 }));
+            }
+        }
+
+        TEST_METHOD(Pivot)
+        {
+            Terminal term;
+            DummyRenderTarget emptyRT;
+            term.Create({ 100, 100 }, 0, emptyRT);
+
+            // Step 1: Create a selection
+            {
+                // (10,10) to (20, 10)
+                term.SelectNewRegion({ 10, 10 }, { 20, 10 });
+
+                // Validate selection area
+                ValidateSingleRowSelection(term, SMALL_RECT({ 10, 10, 20, 10 }));
+            }
+
+            // Step 2: Pivot to (5,10)
+            {
+                term.SetSelectionEnd({ 5, 10 });
+
+                // Validate selection area
+                // NOTE: Pivot should be (10, 10)
+                ValidateSingleRowSelection(term, SMALL_RECT({ 5, 10, 10, 10 }));
+            }
+
+            // Step 3: Pivot back to (20,10)
+            {
+                term.SetSelectionEnd({ 20, 10 });
+
+                // Validate selection area
+                // NOTE: Pivot should still be (10, 10)
+                ValidateSingleRowSelection(term, SMALL_RECT({ 10, 10, 20, 10 }));
+            }
+
+            // Step 4: Pivot back to (5,10) with Shift+Click

```

Comment:
nit: shift+click is not a concept at the pivot level. consider coming up w/ a more succinct way to describe it :smile:

Reply:
This is not the direction I was hoping that these comments would go lol

_shift-click means nothing to the pivot function_. It's a unit test, there's no clicking

Reply:
you ended up deleting the word _pivot_ instead of the word _shift+click_. It doesn't really matter, and it's a minor change, so i'm not gonna blokk